README.md

```diff
@@ -61,6 +61,11 @@ npm install
 sudo npm install -g grunt-cli
 ```
 
+### Build
+```
+grunt webpack
+```
+
```

Comment:
?

Reply:
Adding an instruction to build?

Reply:
For building the `css` file to test locally, you can use the `grunt css` and for building the `js` file locally, you can use `grunt js`.

Reply:
Yep.

Reply:
Updated the readme with proper commands.